src/attackHandler.ts

```diff
@@ -0,0 +1,48 @@
+import { Character } from './character';
+import { WeaponCollision as collision } from './weaponCollision';
+
+export class AttackHandler {
+  static handleAttack(attacker: Character) {
```

Comment:
Consider making this `canAttack(attacker: Character): boolean`. This should return true if the given Character can attack.

Reply:
Or just inline into the other function.

Reply:
I combined to the two functions. I moved the conditional from handleAttack to update and renamed the function to attackHandler.